django_freeradius/settings.py

```diff
@@ -33,3 +33,4 @@
                         'rest_framework.authtoken'}.issubset(settings.INSTALLED_APPS)
 DISPOSABLE_USER_TOKEN = getattr(settings, 'DJANGO_FREERADIUS_DISPOSABLE_USER_TOKEN', True)
 API_ACCOUNTING_AUTO_GROUP = getattr(settings, 'DJANGO_FREERADIUS_API_ACCOUNTING_AUTO_GROUP', True)
+CUSTOM_TYPES_EXTRA = getattr(settings, 'CUSTOM_TYPES_FOR_NAS', None)
```

Comment:
change this to:

```python
EXTRA_NAS_TYPES = getattr(settings, 'DJANGO_FREERADIUS_EXTRA_NAS_TYPES', tuple())
```

Adapt the naming in the other files in order to be consistent with this suggesiton.
Note: default to empty tuple so we can avoid the `if` in `models.py`.